Sources/Signal.swift

```diff
@@ -557,6 +557,16 @@ extension Signal {
 	public func map<U>(_ transform: @escaping (Value) -> U) -> Signal<U, Error> {
 		return flatMapEvent(Signal.Event.map(transform))
 	}
+	
+	/// Map each value in the signal to a new constant value.
+	///
+	/// - parameters:
+	///   - value: A new value.
+	///
+	/// - returns: A signal that will send new values.
+	public func map<U>(to value: U) -> Signal<U, Error> {
```

Comment:
Can you add this to `SignalProducer` too?

Reply:
oh, I forgot
will add it.